s2/builder_snapper.go

```diff
@@ -504,12 +489,14 @@ func (sf IntLatLngSnapper) MinEdgeVertexSeparation() s1.Angle {
 	//    bound approaches 0.5 * snapRadius as the snap radius becomes large
 	//    relative to the grid spacing.
 	vertexSep := sf.MinVertexSeparation()
-	return maxAngle(s1.Angle(1/math.Sqrt(13))*s1.Degree*sf.to,
-		maxAngle((2.0/9.0)*sf.snapRadius, 0.5*(vertexSep/sf.snapRadius)*vertexSep))
+	return maxAngle(0.277*s1.Degree*sf.to, // 1/sqrt(13) in the plane
+		maxAngle(0.222*sf.snapRadius, // 2/9 in the plane
+			0.5*(vertexSep/sf.snapRadius)*vertexSep))
 }
 
 // SnapPoint returns a candidate snap site for the given point.
 func (sf IntLatLngSnapper) SnapPoint(point Point) Point {
+	// ABSL_DCHECK_GE(exponent_, 0);// Make sure snap function was initialized.
```

Comment:
I'm not sure what to make of this comment.

Reply:
finished cleaning up comment.